src/backend-gdix.h

```diff
@@ -74,6 +74,8 @@ struct gdix_vtable_tag {
     int (WINAPI* fn_SetPixelOffsetMode)(dummy_GpGraphics*, dummy_GpPixelOffsetMode);
     int (WINAPI* fn_SetSmoothingMode)(dummy_GpGraphics*, dummy_GpSmoothingMode);
     int (WINAPI* fn_TranslateWorldTransform)(dummy_GpGraphics*, float, float, dummy_GpMatrixOrder);
+    int (WINAPI* fn_SetWorldTransform)(dummy_GpGraphics*, dummy_GpMatrix*, dummy_GpMatrixOrder);
```

Comment:
`GdipSetWorldTransform()` takes only two params. Consult `<gdiplusflat.h>`. Maybe you want `GdipMultiplyWorldTransform()` instead?

Reply:
Initially the argument was just too much. With the requirement for the initial rtl transformation it must be the mentioned matrix now.